packages/web-components/src/components/va-textarea/va-textarea.tsx

```diff
@@ -187,57 +210,78 @@ export class VaTextarea {
         'usa-character-count__status': charcount,
         'usa-character-count__status--invalid': charcount && maxlength && value?.length > maxlength
       });
+      const isFormsPattern = useFormsPattern === 'single' || useFormsPattern === 'multiple' ? true : false;
+      let formsHeading = null;
+      if (isFormsPattern) {
+        const HeaderLevel = getHeaderLevel(formHeadingLevel);
+        formsHeading = (
+          <Fragment>
+            {formHeading &&
+              <HeaderLevel id="form-question" part="form-header">
+                {formHeading}
+              </HeaderLevel>
+            }
+            <div id="form-description">
+              <slot name="form-description"></slot>
```

Comment:
This new named slot `form-description` will be really great to demontrate in the Storybook example. We will also need to make sure it works with screenreaders. Can you also add some content to the Storybook example for it? Similar to text input:

https://design.va.gov/storybook/?path=/docs/uswds-va-text-input--default#forms-pattern-single-error

![Screenshot 2024-01-05 at 8 51 46 AM](https://github.com/department-of-veterans-affairs/component-library/assets/872479/b5cf4a8d-cea2-48e9-894b-280aefeca439)

Reply:
What we need to verify is that when the input is tabbed into, all of that slot content as well as the label (header) and hint text are read by the screenreader:


![Screenshot 2024-01-05 at 8 55 27 AM](https://github.com/department-of-veterans-affairs/component-library/assets/872479/9142336a-0c45-46c6-9a6e-efd1296b0004)